packages/sui-js/README.md

```diff
@@ -0,0 +1,17 @@
+# sui-js
+> Set of useful js utilities
+
+```sh
+$ npm install @s-ui/js --save-dev
```

Comment:
This is a PROD dependency, not DEV. So it should be just `--save`.

Reply:
right. Thanks!